.gitignore

```diff
@@ -146,6 +146,3 @@ typings/
 
 # End of https://www.gitignore.io/api/node,jetbrains+all
 #
-
-## CUSTOM
-dist/
```

Comment:
Any reason this was removed? We probably don't need to have version control on our build output.